src/databricks/labs/ucx/source_code/linters/spark_connect.py

```diff
@@ -96,7 +96,7 @@ def _lint_sc(self, node: NodeNG) -> Iterator[Advice]:
             return
         if node.func.attrname not in self._SC_METHODS:
             return
-        function_name = Tree.get_full_function_name(node)
+        function_name = TreeHelper.get_full_function_name(node)
```

Comment:
don't unnecessarily introduce new public class

Reply:
This is to avoid linting error with public methods > 20